tests/fast-integration/tracing/index.js

```diff
@@ -0,0 +1,70 @@
+const {
+  getEventingBackend,
+  waitForNamespace,
+  switchEventingBackend,
+} = require('../utils');
+const {
+  sendLegacyEventAndCheckTracing,
+  sendCloudEventStructuredModeAndCheckTracing,
+  sendCloudEventBinaryModeAndCheckTracing,
+  waitForSubscriptionsTillReady,
+} = require('../test/fixtures/commerce-mock');
+const {prometheusPortForward} = require('../monitoring/client');
+
+const testNamespace = `test`;
+const mockNamespace = process.env.MOCK_NAMESPACE || 'mocks';
+const backendK8sSecretName = process.env.BACKEND_SECRET_NAME || 'tracing-backend';
+const backendK8sSecretNamespace = process.env.BACKEND_SECRET_NAMESPACE || 'default';
+const isSKR = process.env.KYMA_TYPE === 'SKR';
+const natsBackend = 'nats';
+
+async function tracingTests() {
+  if (isSKR) {
+    console.log('Skipping eventing tracing tests on SKR...');
+    return;
+  }
+  describe('Tracing Tests:', function() {
+    this.timeout(5 * 60 * 1000); // 5 min
+    let cancelPrometheusPortForward = null;
+
+    before('Ensure the test and mock namespaces exist', async function() {
+      await waitForNamespace(testNamespace);
+      await waitForNamespace(mockNamespace);
+      cancelPrometheusPortForward = prometheusPortForward();
+    });
+
+    after(async function() {
+      cancelPrometheusPortForward();
+    });
+
+    // Run Eventing tracing tests
+    context('with Nats backend', function() {
+      it('Switch Eventing Backend to Nats', async function() {
```

Comment:
Do we ever switch to BEB in any of the tests where we use this tracing tests? IMO, the backend would be already in NATS. But confirm it please.